db/migrations/20240523181636_userTable.sql

```diff
@@ -4,4 +4,4 @@ CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
 CREATE TABLE Users (
   user_id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
   user_name VARCHAR(255) NOT NULL UNIQUE
-);
+);
```

Comment:
Ahh I think this might be why you had that error about an existing migration having changed!

And in general I think we want the new line at the end of this file so probably good to change this back.
But that might make you drop and re-create the DB again since this migration hash will change again lol

Reply:
Bump on this! I think we want the trailing new line back here